doc/reference.md

```diff
@@ -97,7 +117,7 @@ For maven you must configure the surefire plugin for junit tests.
 </plugin>
 ```
 
-And then add the Kotest JUnit5 runner to your build.
+And then add the Kotest JUnit5 runner to your build to use the framework product.
```

Comment:
I think we want to recommend the console runner even more than JUnit5, perhaps we should suggest it?

Reply:
The console runner can't be used for gradle, only for the plugin. We can clarify once we release the plugin.